include/Mixer.h

```diff
@@ -151,38 +151,38 @@ class LMMS_EXPORT FxMixer : public Model, public JournallingObject
 
 	QString nodeName() const override
 	{
-		return "fxmixer";
+		return "mixer";
 	}
 
-	FxChannel * effectChannel( int _ch )
+	MixerChannel * effectChannel( int _ch )
```

Comment:
Should all "effectChannel", "EffectChannel" and "effect-channel" from "src/..." be changed to "mixerChannel" etc.?